test/CoinbaseSmartWalletFactory.t.sol

```diff
@@ -32,6 +33,20 @@ contract CoinbaseSmartWalletFactoryTest is Test {
         factory.createAccount{value: 1e18}(owners, 0);
     }
 
+    function test_exitIfAccountIsAlreadyInitialized() public {
```

Comment:
I'm not sure we really test this here 🤔

Reply:
So the `test_exitIfAccountIsAlreadyInitialized` was intended to catch the branch that foundry coverage was missing. In messing with MultiOwnable, I noticed that this test ticked the coverage for Factory. I think that the difference between the impl. here and the one in MultiOwnable is that the actual factory is included.